enclave_wrangler/objects_api.py

```diff
@@ -789,7 +789,7 @@ def refresh_tables_for_object():
 
 
 def get_concept_set_version_expression_items(
-    version_id: Union[str, int], return_detail='full', handle_paginated=False
+    version_id: Union[str, int], return_detail=['id','full'][1], handle_paginated=False
```

Comment:
I messed around with this in the previous commit as well. I ended up leaving it this way, since I had changed the default I went through all the calls of this function and made sure that they would output the right version for their context.